### PR TITLE
Fix #1920 Converted MetaLinkNetworkEntityTest to Kotlin

### LINKED ISSUE #1920: Convert MetaLinkNetworkEntityTest to Kotlin
**Is your feature request related to a problem? Please describe.**

MetaLinkNetworkEntityTest is in java. It can be converted to Kotlin.

**Describe the solution you'd like**

Convert it to kotlin.

**Additional context**

Parent issue #1826

### PR DESCRIPTION
**Pull request by akkywadhwa**
Fixes #1920 

Converted it to Kotlin.

**Screenshots** 
image

### CODE REVIEW

**Comment by yashk2000** [comment]
@akkywadhwa please check the pr properly. Your build is failing. 

```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/runner/work/kiwix-android/kiwix-android/buildSrc/build.gradle.kts' line: 1

* What went wrong:
Plugin [id: 'org.gradle.kotlin.kotlin-dsl', version: '1.3.3'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'org.gradle.kotlin.kotlin-dsl:org.gradle.kotlin.kotlin-dsl.gradle.plugin:1.3.3')
  Searched in the following repositories:
    Gradle Central Plugin Repository
```

File: core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        ),
+        url(
+          "de", 4,
+          "http://mirror.netcologne.de/kiwix/zim/wikipedia/wikipedia_af_all_nopic_2016-05.zim"
+        ),
+        url("fr", 5, "http://mirror3.kiwix.org/zim/wikipedia/wikipedia_af_all_nopic_2016-05.zim")
+      )
+    )
+    // Basic file attributes
+    Assert.assertThat(
```
Comment (macgills) [inline core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt]: Please update these `assertThat` to use `assertJ` eg
```
assertThat(result.getFile().getName()).isEqualTo("wikipedia_af_all_nopic_2016-05.zim")
```

**Comment by akkywadhwa** [comment]
@macgills I ran "gradle :core:detekt" locally and it ran successfully with no error (Refer screenshot below).

image

But when I pushed the code, "gradle :core:detekt" failed with the following errors: 

`> Task :core:ktlintTestSourceSetCheck
/home/runner/work/kiwix-android/kiwix-android/core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt:37:1: Needless blank line(s)
/home/runner/work/kiwix-android/kiwix-android/core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt:48:1: Exceeded max line length (100) (cannot be auto-corrected)`

I have faced the same issue while making other PRs too. Can you please help me to solve this?

**Comment by macgills** [comment]
```
> Task :core:ktlintTestSourceSetCheck FAILED
/home/runner/work/kiwix-android/kiwix-android/core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt:37:1: Needless blank line(s)
FAILURE: Build failed with an exception.
/home/runner/work/kiwix-android/kiwix-android/core/src/test/java/org/kiwix/kiwixmobile/core/entity/MetaLinkNetworkEntityTest.kt:48:1: Exceeded max line length (100) (cannot be auto-corrected)
```

For you to be able to commit these files you would have had to do one of the following
1. Bypass commit hooks
1. Never have installed the git hooks (done whenever the app is built)

Neither scenario is ideal, please fix these formatting errors

**Pull request merged**